WalletWasabi.Fluent.Generators/WalletWasabi.Fluent.Generators.csproj

```diff
@@ -5,6 +5,12 @@
 		<AnalysisLevel>latest</AnalysisLevel>
 		<Nullable>enable</Nullable>
 		<LangVersion>latest</LangVersion>
+		<EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+		<CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)\GeneratedFiles</CompilerGeneratedFilesOutputPath>
```

Comment:
These are taken from https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Fluent/WalletWasabi.Fluent.csproj

The rest is taken from https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/WalletWasabi.Gui.csproj